src/test/groovy/sdkman/support/SdkmanEnvSpecification.groovy

```diff
@@ -11,7 +10,6 @@ abstract class SdkmanEnvSpecification extends BashEnvSpecification {
 	SdkmanBashEnvBuilder sdkmanBashEnvBuilder
 
 	CurlStub curlStub
-	UnameStub unameStub
```

Comment:
It feels like this should be here because it is a base class for all the specs. wdyt?